templates/smb.conf.client.j2

```diff
@@ -9,6 +9,11 @@
 
 	idmap config * : range = 10000-20000000
 	idmap config * : backend = tdb
+{% if samba_log_level is defined %}
+	log level = {{ samba_log_level }}
+	timestamp logs = yes
```

Comment:
I don't think that defining `samba_log_level` should set some another parameters. As a user of playbook I expect that `samba_log_level` will set only `log level` and nothing more. It should be renamed to something more expectable. E.g. `samba_verbose_logging` or so. Or we should make dedicated variables for logging parameters.

Reply:
> I expect that samba_log_level will set only log level and nothing more.

The log should have contained the timestamps in first place, without having to set any extra parameters. And those timestamps should have been be accurate enough (at least millisecond resolution) without having to set even more parameter(s).

Thus your expectation is not reasonable at all